tests/test_language_extract_go.py

```diff
@@ -0,0 +1,117 @@
+import pathlib
+
+import pytest
+
+from floss.results import StaticString, StringEncoding
+from floss.language.go.extract import extract_go_strings
+
+
+@pytest.fixture(scope="module")
+def go_strings32():
+    n = 6
+    path = pathlib.Path(__file__).parent / "data" / "src" / "go-hello" / "bin" / "go-hello.exe"
+    return list(extract_go_strings(path, n))
+
+
+@pytest.fixture(scope="module")
+def go_strings64():
+    n = 6
+    path = pathlib.Path(__file__).parent / "data" / "src" / "go-hello" / "bin" / "go-hello64.exe"
+    return list(extract_go_strings(path, n))
+
+
+@pytest.mark.parametrize(
+    "string,offset,encoding,go_strings",
+    [
+        # .data:00534944 A0 35 4A 00                       dd offset aAdaptivestacks ; "adaptivestackstart"
+        # .data:00534948 12                                db  12h
+        pytest.param("adaptivestackstart", 0x534944, StringEncoding.ASCII, "go_strings32"),
+        # .data:00534944 A0 35 4A 00                       dd offset aAdaptivestacks ; "adaptivestackstart"
+        # .data:00534948 12                                db  12h
+        pytest.param("adaptivestackstart", 0x541568, StringEncoding.ASCII, "go_strings64"),
+    ],
+)
+def test_data_string_offset(request, string, offset, encoding, go_strings):
+    # .data:0000000000541568 33 A8 4A 00 00 00…                dq offset aAdaptivestacks ; "adaptivestackstart"
+    # .data:0000000000541570 12                                db  12h
+    assert StaticString(string=string, offset=offset, encoding=encoding) in request.getfixturevalue(go_strings)
+
+
+@pytest.mark.parametrize(
+    "string,offset,encoding,go_strings",
+    [
+        # .text:0048B12F 8D 05 F8 08 4C 00 lea     eax, off_4C08F8 ; "hello world"
+        # .text:0048B135 89 44 24 24       mov     [esp+28h+var_4], eax
+        pytest.param("hello world", 0x4C08F8, StringEncoding.ASCII, "go_strings32"),
```

Comment:
this is inconsistent right now (file offset vs. RVA), I think this one is good but the others need to be updated